src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -1,9 +1,19 @@
 export interface MenuVariables {
   color: string
+
+  primaryActiveColor: string
+  primaryFocusedColor: string
+
+  backgroundColor: string
 }
 
 export default (siteVars: any): MenuVariables => {
   return {
     color: siteVars.white,
+
+    primaryActiveColor: siteVars.black,
+    primaryFocusedColor: siteVars.black,
+
+    backgroundColor: siteVars.black,
   }
 }
```

Comment:
Why do we not need color scheme in this file, but we do for the contrast variables?

Reply:
Before adding any more variables, let's nail down exactly why we need certain variables and make them consistent across components.   To do this, we need user stories and requirements to fulfill.

We should probably have a focused effort on reviewing components and the usefulness of certain variables across many components.  Once we have requirements and have spent some time designing a variable scheme to fill the requirements, we can then start adding more variables.